domains_test.go

```diff
@@ -272,12 +281,14 @@ func TestDomains_EditRecordForDomainName(t *testing.T) {
 	setup()
 	defer teardown()
 
+	port := 10
+
 	editRequest := &DomainRecordEditRequest{
 		Type:     "CNAME",
 		Name:     "example",
 		Data:     "@",
 		Priority: 10,
-		Port:     10,
+		Port:     &port,
```

Comment:
Can we have a test case here where the value of `Port` is `0`? Or is it not necessary to test?

Reply:
I don't think it's needed now that changed to a pointer but i can totally add it 💃

Reply:
What @YingCGooi said. Let's add that test case

Reply:
done.